### DISCUSSION TITLE
upgrade 13

### DISCUSSION THREAD

**Discussion by tnwss** [baughmann/designaire-z390-intel-i9-9900k-opencore#71] [General]
https://user-images.githubusercontent.com/22867489/198817393-1206a079-a78b-4a52-b844-db3e8f27ca14.mp4

upgrade 13  reboot

**Comment by seven-of-eleven** [comment]
Not sure what's going on. Looks like an issue with the EFI folder. I'll be updating the Repo over the next week to the latest release that may help you. I did upgrade to MacOS13 myself with OC 0.8.5 and everything is working. I had to download the complete installer for it to work. If you can provide more details, EFI folder, or logs, then I may be able to help more.
